datasets/arabic_pos_dialect/README.md

```diff
@@ -0,0 +1,206 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- found
+languages:
+- ar
+licenses: [apache-2.0]
+multilinguality:
+- multilingual
+size_categories:
+- 1K<n<10K
+source_datasets:
+- extended
+task_categories:
+- text-classification
+task_ids:
+- multi-class-classification
```

Comment:
Maybe specify POS tagging or something here ? I don't think it's in the taxonomy yet so we should add it